grammars/ruby.cson

```diff
@@ -351,7 +351,8 @@
   }
   {
     'match': '''(?x)
-      \\b(
+      (\\b|(?<!\\d[ \\t]|\\d)-)
```

Comment:
Not equivalent this way. You have to tell the parser this line ain't over by placing a backslash at the end:

```Ruby
3 \
-4
```

Yet the rule still breaks. `3  -4` also breaks it. But seriously, for a person who writes code this way, I think they wouldn't care. 😂  Could you show me where did the problems happen?

On second thought, `(\\b|(?<!\\w|\\w[ \\t])-)` is better, or just `(\\b|(?<!\\w)-)`, if we are still gonna keep this commit.